app/models/spree/store_decorator.rb

```diff
@@ -1,24 +1,30 @@
 module Spree
-  Store.class_eval do
-    has_and_belongs_to_many :products, join_table: 'spree_products_stores'
-    has_many :taxonomies
-    has_many :orders
+  module StoreDecorator
+    def self.prepended(base)
+      base.has_and_belongs_to_many :products, join_table: 'spree_products_stores'
+      base.has_many :taxonomies
+      base.has_many :orders
+      base.has_many :store_payment_methods
+      base.has_many :payment_methods, through: :store_payment_methods
+      base.has_many :store_shipping_methods
+      base.has_many :shipping_methods, through: :store_shipping_methods
+      base.has_and_belongs_to_many :promotion_rules, class_name: 'Spree::Promotion::Rules::Store', join_table: 'spree_promotion_rules_stores', association_foreign_key: 'promotion_rule_id'
 
-    has_many :store_payment_methods
-    has_many :payment_methods, through: :store_payment_methods
+      if Spree.version.to_f < 3.7
+        base.has_attached_file :logo,
```

Comment:
I think this will conflict with active storage. But we are not using the logo. So maybe It should be removed

Reply:
@balazslaszlo I will remove this code